packages/form-js/test/spec/FormEditor.spec.js

```diff
@@ -8,6 +8,10 @@ import schema from './form.json';
 
 import { insertStyles } from '../TestHelper';
 
+import {

```

Comment:
And here?

Reply:
Looks like an import to me :monkey_face: